scripts/lint.sh

```diff
@@ -107,6 +123,7 @@ if  [[
     echo -e "pydocstyle on tests exit: $pydocstyle_test_exit_code"
     echo -e "pylint exit: $pylint_exit_code"
     echo -e "bandit exit: $bandit_exit_code"
+    echo -e "sphinx-build exit: $sphinx_build_exit_code"
```

Comment:
Either move this up one or move the sphinx check to after the bandit check. Let's keep these lists of commands parallel.